src/rgw/rgw_rest.cc

```diff
@@ -1139,18 +1138,17 @@ int RGWPutObj_ObjStore::get_data(bufferlist& bl)
 
   int len = 0;
   if (cl) {
-    ACCOUNTING_IO(s)->set_account(true);
+    /* There is no need to manage the accounting enablement here. It's now
```

Comment:
@amitkumar50 to which guidelines or pr are you referring? i only see https://github.com/ceph/ceph/pull/16705, which is only about ceph docs - not code comments